src/biconnectivity/biconnect.jl

```diff
@@ -31,30 +34,30 @@ function visit!(g::AbstractGraph, state::Biconnections, u::Integer, v::Integer)
     for w in outneighbors(g, v)
         if state.depth[w] == 0
             children += 1
-            push!(state.stack, Edge(min(v, w), max(v, w)))
+            push!(state.stack, E(min(v, w), max(v, w)))
             visit!(g, state, v, w)
             state.low[v] = min(state.low[v], state.low[w])
 
             #Checking the root, and then the non-roots if they are articulation points
             if (u == v && children > 1) || (u != v && state.low[w] >= state.depth[v])
-                e = Edge(0, 0)  #Invalid Edge, used for comparison only
-                st = Vector{Edge}()
-                while e != Edge(min(v, w), max(v, w))
+                e = E(0, 0)  #Invalid Edge, used for comparison only
+                st = Vector{E}()
+                while e != E(min(v, w), max(v, w))
                     e = pop!(state.stack)
                     push!(st, e)
                 end
                 push!(state.biconnected_comps, st)
             end
 
         elseif w != u && state.low[v] > state.depth[w]
-            push!(state.stack, Edge(min(v, w), max(v, w)))
+            push!(state.stack, E(min(v, w), max(v, w)))
             state.low[v] = state.depth[w]
         end
     end
 end
 
 """
-    biconnected_components(g)
+    biconnected_components(g) -> Vector{Vector{Edge{eltype(g)}}}
```

Comment:
If you're going to document the return type, use `biconnected_components(g))::Vector{...}` here. The `->` is used for lambdas.

Reply:
The convention with `->` for documenting the return type is used in Julia code though - see for example the function `count`.

Reply:
I stand corrected. We should go through and make sure we're consistent.